### PR TITLE
added buttons 

### PR DESCRIPTION
**Pull request by josephos416**

### CODE REVIEW

File: bbreadfront/src/ProfileForm.js [hunk 1]
```diff
@@ -22,7 +22,11 @@ function ProfileForm(props) {
   }
 
   return (
-      null
+    <div><input type="button" value="Login" onClick={props.addFriend} />
```
Comment (hannahtobiason) [inline bbreadfront/src/ProfileForm.js:25]: Maybe think about changing the value here to a more descriptive name than Login, as we are not logging in.

**Review by hannahtobiason — APPROVED** [review]

File: bbreadfront/src/ProfileForm.js (hunk 1 above)
Comment (Eriken79) [reply bbreadfront/src/ProfileForm.js:25]: agreed, as far as the merge conflict goes for this part of the code, I think the <div> tag containing the buttons can just be added alongside the button that redirects to the new grocery run page. Not a big conflict

**Comment by Eriken79** [comment]
Looks good, once the put is completely integrated into the profile page, we'll be very close to being done with the main functionality for the project, good work Joseph.

**Comment by Eriken79** [comment]
Sorry, I just noticed this, what are the 3,000 lines that we're changed? Is that just .json file weirdness going on?

**Comment by MohamedCheour98** [comment]
Alright. Great work Joseph. 

I tried to fix the merge conflict. But I didn't want to mess with anything that you have worked on. So, if I wasn't sure about something, I made sure not to touch it. Now, the only reason why the front CI build is not passing is because of a merge conflict in the ProfileForm file. Make sure to take a look at that whenever you can and fix the conflict. 

Thanks again.

**Pull request closed without merging**